MailCore/Models/Thread.swift

```diff
@@ -125,6 +125,10 @@ public class Thread: Object, Decodable, Identifiable {
         return messages.toArray() + duplicates.toArray()
     }
 
+    public var isSnoozed: Bool {
+        snoozeState == .snoozed && snoozeEndDate != nil && snoozeUuid != nil
```

Comment:
Out of curiosity. Is it possible for a snoozed message (snoozeState == .snoozed) to not have a snoozeEndDate ?